spark3/src/main/java/org/apache/iceberg/spark/procedures/ExpireSnapshotsProcedure.java

```diff
@@ -79,6 +85,7 @@ public StructType outputType() {
     Identifier tableIdent = toIdentifier(args.getString(0), PARAMETERS[0].name());
     Long olderThanMillis = args.isNullAt(1) ? null : DateTimeUtil.microsToMillis(args.getLong(1));
     Integer retainLastNum = args.isNullAt(2) ? null : args.getInt(2);
+    Integer maxConcurrentDeletes = args.isNullAt(3) ? null : args.getInt(3);
 
```

Comment:
I think the only thing left is we should check that maxCurrentDeletes is null or > 0, after that I think we are good to merge